src/Promitor.Agents.ResourceDiscovery/Configuration/AzureLandscape.cs

```diff
@@ -8,5 +11,7 @@ public class AzureLandscape
         public string TenantId { get; set; }
         public List<string> Subscriptions { get; set; }
         public AzureCloud Cloud { get; set; } = AzureCloud.Global;
+        [JsonPropertyName("endpoints")]
+        public AzureEnvironment? AzureEnvironment { get; set; }
```

Comment:
Let's stick with using Cloud by default and not mutate in validation logic; but when endpoints is specified us that over cloud and its respective environment